CODEOWNERS

```diff
@@ -5,6 +5,8 @@
 ctapipe/calib/camera  @watsonjj
 ctapipe/image/extractor.py  @watsonjj @HealthyPear
 
+ctapipe/monitoring  @TjarkMiener
```

Comment:
we don't really make use of this mechanism and the people listed here are not currenly active anymore, so I'd rather just remove it (in another PR) than adding to it here.

Reply:
Okay, I removed my entry.